models/models.js

```diff
@@ -22,4 +22,28 @@ const selectArticleById = (article_id) => {
         })
     }
 
-module.exports = { selectTopics, selectArticleById }
\ No newline at end of file
+const selectAllArticles = (orderBy) => {
+    console.log('model 1')
+    return db
+        .query(`
+            SELECT
+            articles.author, 
```

Comment:
This can use articles.* instead of naming each column individually.

Reply:
I did them individually so to exclude body

Reply:
ah i understand! This might be a change to sprint! Feel free to ignore.